controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -51,13 +51,15 @@ import (
 )
 
 const (
-	kubeProxyKey              = "kube-proxy"
-	kubeadmConfigKey          = "kubeadm-config"
-	kubeletConfigKey          = "kubelet"
-	cgroupDriverKey           = "cgroupDriver"
-	labelNodeRoleControlPlane = "node-role.kubernetes.io/master"
-	clusterStatusKey          = "ClusterStatus"
-	clusterConfigurationKey   = "ClusterConfiguration"
+	kubeProxyKey     = "kube-proxy"
+	kubeadmConfigKey = "kubeadm-config"
+	kubeletConfigKey = "kubelet"
+	cgroupDriverKey  = "cgroupDriver"
+	// Deprecated: https://github.com/kubernetes/kubeadm/issues/2200
+	labelNodeRoleOldControlPlane = "node-role.kubernetes.io/master"
```

Comment:
Move the comment on the side? Otherwise it seems the whole block is deprecated

Reply:
Done